pkgs/development/python-modules/tensorflow/bin.nix

```diff
@@ -73,7 +75,7 @@ buildPythonPackage {
     protobuf
     numpy
     scipy
-    jax
+    (if !cudaPackages.cudaFlags.isJetsonBuild then jax else ml-dtypes)
```

Comment:
Maybe `if B then y else x` instead of `if !A then x else y`?

Reply:
Done